### ISSUE TITLE
UX on 2FA select challenge page is broken

### ISSUE THREAD

**Issue by heapifyman** [nextcloud/twofactor_webauthn#335]
### Steps to reproduce
1. Setup 2FA with both Authenticator App and a security key (for example SoloKey)
2. Login

### Expected behaviour
The page after the login form `/login/selectchallenge` should be usable with keyboard.

The "buttons" to select the 2FA method (either Authenticator App or Security Key) should be highlighted when using `Tab` key to switch focus between elements on the page.

The buttons (form) should have a visual appearance that is consistent with the rest of the default nextcloud style.

### Actual behaviour
The "buttons" to select the 2FA method (either Authenticator App or Security Key) are *not* highlighted when using `Tab` key to switch focus between elements on the page.

It is impossible to know which button (method) is currently selected.

This was working in nextcloud 24.

The buttons look terrible, don't follow the default nextcloud style for buttons - see 
![swappy-20230122_120616](https://user-images.githubusercontent.com/274236/213913004-4679658d-c0a0-4f2f-b46c-92231398f5d4.png)

### Server configuration

**Operating system**:

**Web server:**

**Database:**

**PHP version:**

**Version:** (see admin page)

**Updated from an older version or fresh install:** updated from 24

**List of activated apps:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your server installation folder
```

**The content of config/config.php:**
```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or

Insert your config.php content here
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```

#### Client configuration
**Browser:** Firefox and Brave

**Operating system:** Manjaro Linux

##### Web server error log
```
Insert your webserver log here
```

##### Server log (data/nextcloud.log)
```
Insert your server log here
```

##### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
```

**Comment by st3iny** [comment]
This has been fixed in the meantime.

**Comment by st3iny** [comment]
You're right, the tabbing is not fixed yet. I'm sorry for closing the ticket early.
